matchms/Pipeline.py

```diff
@@ -325,7 +354,7 @@ def write_to_logfile(self, line):
             with open(self.logging_file, "a", encoding="utf-8") as f:
                 f.write(line + '\n')
 
-    def import_data(self, query_files, reference_files=None):
+    def import_spectrums(self, query_files, reference_files=None):
```

Comment:
Some of the functionality of this function could be moved to the general load or import spectra function, such as the error handling etc.

Reply:
That certainly makes sense, I have moved most of the functionality to load_spectrums